sdk/textanalytics/ai-text-analytics/src/util.ts

```diff
@@ -2,7 +2,6 @@
 // Licensed under the MIT license.
 
 import { RestError } from "@azure/core-rest-pipeline";
-import { URL, URLSearchParams } from "./utils/url";
```

Comment:
should we delete the code for `./utils/url`?

Reply:
Right! Deleted.